c-deps/libroach/engine.cc

```diff
@@ -171,6 +172,7 @@ DBStatus DBImpl::ApplyBatchRepr(DBSlice repr, bool sync) {
   rocksdb::WriteBatch batch(ToString(repr));
   rocksdb::WriteOptions options;
   options.sync = sync;
+  options.disableWAL = skip_wal;
```

Comment:
If we're going to expose the option in the `RocksDBConfig` then it should probably apply to all operations, not just `ApplyBatchRepr`.